firestore/cloud-client/snippets_test.py

```diff
@@ -255,3 +255,16 @@ def test_listen_for_changes(capsys):
 
 def test_delete_full_collection():
     snippets.delete_full_collection()
+
+
+@pytest.mark.skip(reason="Dependant on a composite index being created,"
```

Comment:
I'm still not sure why a skip is being used here. Why can't we integrate the creation of the index into the tests? 

More so, why doesn't the snippet show users how to do these things? Is that information found in a different snippet?

Reply:
+1 to @kurtisvg, if it hinders the usage of the sample then it should be included. 

If it's a timing issue given I/O starts tomorrow, then I'm okay with approving the sample as long as it's fixed afterwards.

Reply:
Creation of the necessary composite index is something that today has to be done through the UI or the Firestore cli. After I/O I'm going to be running through each of the silver languages to make sure there are firestore admin clients and then work on using those to integrate the creation of the index into the setup of the test run.

Reply:
Would you please file a bug for this and then add it as `# TODO: b/####` to provide additional context for future readers?

Reply:
Bug created, and TODO added.